misc/build_helpers/build-manylinux-wheels.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+# This runs in the container to actually build the wheels.
+BUILDER="/io/misc/build_helpers/_build-wheels.sh"
+
+# This image can build x86_64 (64 bit) manylinux wheels.
+DOCKER_IMAGE="quay.io/pypa/manylinux1_x86_64"
```

Comment:
Alternatively, we could suggest building the Docker images locally from the Dockerfiles on github - eg https://github.com/pypa/manylinux/blob/master/docker/Dockerfile-x86_64